api/models/formation.go

```diff
@@ -118,6 +121,14 @@ func SetFormation(app, process string, count, memory int64) error {
 		params[fmt.Sprintf("%sMemory", UpperName(process))] = fmt.Sprintf("%d", memory)
 	}
 
+	if cpu > 0 {
+		if cpu > capacity.InstanceCPU {
+			return fmt.Errorf("requested cpu %d greater than instance size %d", cpu, capacity.InstanceCPU)
```

Comment:
I would assume that the service would throw the [`not enough cpu`](http://docs.aws.amazon.com/AmazonECS/latest/developerguide/troubleshooting.html) error